pkg/cmd/admin/router/router.go

```diff
@@ -777,12 +797,21 @@ func validateServiceAccount(client *kclient.Client, ns string, serviceAccount st
 	for _, scc := range sccList.Items {
 		if admission.ConstraintAppliesTo(&scc, userInfo) {
 			switch {
+			case hostPorts && scc.AllowHostPorts:
```

Comment:
shouldn't hostnetwork take priority? (I'm basing that on my reading of the flag help for `--host-ports` which made it sound like it only applied when `--host-network` was false)